opentracing-api/pom.xml

```diff
@@ -29,8 +29,8 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
-        <main.signature.artifact>java16</main.signature.artifact>
+        <main.java.version>1.7</main.java.version>
```

Comment:
I really wanted to use try-with-resources for the `Scheduler.Continuation`. Can someone with more JVM ecosystem experience chime in about how realistic this 1.7 requirement is? cc @pavolloffay @yurishkuro